collect_app/src/main/res/layout/form_chooser_list_item_map_button.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/map_button"
+        style="@style/Widget.MaterialComponents.Button.OutlinedButton.Icon"
+        android:background="?selectableItemBackground"
+        android:focusable="false"
+        android:layout_width="40sp"
+        android:layout_height="40sp"
+        android:insetLeft="0sp"
+        android:insetRight="0sp"
+        android:insetTop="0sp"
+        android:insetBottom="0sp"
+        android:padding="8sp"
+        app:icon="@drawable/ic_map"
+        app:iconTint="@color/blue_500" />
```

Comment:
Outlined buttons should really be using `?attr/colorPrimary` for their text/icons ([specs here](https://material.io/components/buttons/#theming)). This is of course what the button would use if we didn't set a tint. The problem we have right now is that our `colorPrimary` is exactly the same as our `colorSurface` so that wouldn't work. Amazingly the Sketch plugin for Material Design (I've attached the PDF output) works this out and it suggests we should be using a black (I'm guessing it just chooses the darkest version of our `colorPrimary`). I think we probably need to make decisions about our `colorPrimary` in the next release as this is going to get messy if we can't just use defaults from our theme. 

For the moment, could you just use `?attr/colorOnSurface` or `?attr/colorOnPrimary` (they're basically the same). I'll quickly bash out a PR after this with a custom `OutlinedButton` style that all the existing ones in the app can use until we can switch our `colorPrimary` to something useful!
 
[collect-styleguide.pdf](https://github.com/opendatakit/collect/files/3863448/collect-styleguide.pdf)

Reply:
All that sounds great, thank you!